### ISSUE TITLE
Replacing ExpectedException on async method is not adding aync on lambda expression

### ISSUE THREAD

**Issue by rajivsnet** [wachulski/nunit-migrator#32]
Old code
```
[Test]
[ExpectedException(typeof(InvalidCalcException), ExpectedMessage = "Invalid calculation passed.")]
public async Task FailedCalculationThrowsException()
{
      var task = calculator.PerformNewCalculationAsync(new CalculationArguments("DivideAnythingByZero"), CancellationToken.None);
      await task; //throws exception
}
```
After migration
```
        [Test]
        public async Task FailedCalculationThrowsException()
        {
			var ex = Assert.Throws<InvalidCalcException>(() =>
			{
				var task = calculator.PerformNewCalculationAsync(new CalculationArguments("DivideAnythingByZero"), CancellationToken.None);
				await task; //throws exception
			});
			Assert.That(ex.Message, Is.EqualTo("Invalid calculation passed."));
        }
```

But then I am getting 

>This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.

I could solve it using
var ex = Assert.Throws<InvalidCalcException>(**async** () =>

Maybe either the migrator can skip all async methods or it adds the async to lambda expression. I don't know which one would be better idea.

**Comment by wachulski** [comment]
Thanks for raising this issue. Let me see what it's feasible in the next days.

**Comment by wachulski** [comment]
Just fixed, a new version (1.1.2) should be available on nuget in a few minutes.

**Comment by ghost** [comment]
Thank you for the fix. It worked.
